auditwheel/wheeltools.py

```diff
@@ -27,6 +28,16 @@ class WheelToolsError(Exception):
     pass
 
 
+def open_for_csv(name, mode):
```

Comment:
Given that auditwheel only runs on Python 3 and there is only one place where this function is called, we could safely replace that call with `open(name, mode, newline='', encoding='utf-8')`. That avoids the unnecessary version check and makes the code more readable.

Reply:
I introduce this change. Could you restart failed steep. It fail on timeout error.

Reply:
Restarted the ppc64 build.